cumulusci/cli/ui.py

```diff
@@ -243,18 +244,52 @@ def describe(self, sobj_name, detailed=False, format="pprint"):
             >>> describe("Account")
             >>> data = describe("Account", detailed=True, format=obj)
             """
-        from pprint import pprint
-
         data = getattr(self._sf, sobj_name).describe()
 
-        if detailed:
-            rc = data
+        if not detailed:
+            data = [_summarize(field) for field in data["fields"]]
+
+        return self._format(data, format)
+
+    def record_count(self, *sobject_list: str, format="table"):
+        """Get approximate record counts for sObjects
+
+        If the approximate count is 0, this function will return nothing.
+        If the sObject doesn't exist in the org, the same is true
+        (as per the Salesforce REST API)
+
+        Arguments:
+
+            sobject_list - list of sobjects to display, or a single sObject
```

Comment:
It looks like it's also possible to get the counts for all objects by not specifying any sobject_list, but I don't see that mentioned here.